src/test/java/net/apnic/rdapd/rdap/RDAPTestingMatches.java

```diff
@@ -0,0 +1,24 @@
+
+package net.apnic.rdapd.rdap;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+
+public class RDAPTestingMatches
+{
+    public static Matcher<String> isRDAPDateTimeString()
+    {
+        return new BaseMatcher<String>() {
+            @Override
+            public boolean matches(final Object item) {
+                return item == null ? true : ((String)item).matches(
+                    "^[0-9]{4}(-[0-9]{2}){2}T([0-9]{2}:){2}[0-9]{2}Z$");
```

Comment:
Just be aware this allows illegal values. eg 0000-90-90T90:90:90Z
{suggestion} Parsing with a java.time.format.DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'") will catch that, and look better than a regex :)